spec/factories/user.rb

```diff
@@ -5,6 +5,9 @@
     password "buttscarlton"
     password_confirmation "buttscarlton"
     phone_number '617-111-1111'
+    address_line_1 '123 main'
+    address_city 'boston'
+    address_zip_code 02110
```

Comment:
I left state as a non-required-field in case of the edge case of a donor coming from a country that doesn't have states. For US and Canada addresses, the city and zip code should be sufficient to determine state. (Although, for that matter, zip_code should probably be a string, since some places have letters in them)